src/main/java/com/sonyericsson/jenkins/plugins/bfa/PluginImpl.java

```diff
@@ -115,6 +116,7 @@
     private String noCausesMessage;
 
     private Boolean globalEnabled;
+    private Boolean doNotAnalyzeAbortedJob;
```

Comment:
no need for a **B**oolean object here, should just be a **b**oolean type since it defaults to `false`.